cmd/edenStart.go

```diff
@@ -122,6 +124,30 @@ var startCmd = &cobra.Command{
 	},
 }
 
+func getUplinkPortIPMap() map[string]net.IP {
```

Comment:
Does this function belong here? 
It looks it should be wrapped somewhere around set-ports.sh ?

Reply:
I am lost. Please help me here.
This function makes API call to controller for the lastest info message that the device sent.
We read the IP addresses that the edge-node got from DHCP from this reponse. These IP addresses are used for setting up port map rules during the test.
Should this be done using shell script and pass the information to go program somehow?